data/campaigns/Under_the_Burning_Suns/units/quenoth/Flanker.cfg

```diff
@@ -11,41 +11,60 @@
     {TRAIT_RESILIENT}
     image=units/quenoth/flanker.png
     profile="portraits/quenoth/flanker.webp"
-    hitpoints=43
+    hitpoints=45
     movement_type=quenoth_foot
-    movement=5
-    experience=90
+    movement=6
+    experience=70
     level=2
     alignment=lawful
     advances_to=Quenoth Ranger
-    cost=28
+    cost=30
     usage=fighter
     description= _ "Some fighters find that their true talents lie not within rigid spear ranks, but out in the sands, striking at the flanks and rear of enemy forces. Fast and nimble, these elves make use of the weaknesses in enemy formations to disrupt their lines and sow chaos on the battlefield. In more cutthroat conflicts, Flankers even rely on the use of poison — a tool whose use was frowned on by their ancestors — to cripple their foes before charging in for the finish."
     die_sound={SOUND_LIST:ELF_HIT}
     [abilities]
         {ABILITY_SKIRMISHER}
     [/abilities]
+    attacks = 4 
```

Comment:
Stylistically, it seems we prefer not to have spaces around the `=`.

Reply:
Yeah, in case of unit files sure my mistake, I use it mostly because it is way easier to read the code for me with this formating but that mostly applies to abilities and not units that already have different formating.